pandas/tests/test_window.py

```diff
@@ -594,6 +594,20 @@ def test_closed_min_max_minp(self, func, closed, expected):
         expected = pd.Series(expected, index=ser.index)
         tm.assert_series_equal(result, expected)
 
+    @pytest.mark.parametrize("closed,expected", [
+        ('right', [0, 0.5, 1, 2, 3, 4, 5, 6, 7, 8]),
```

Comment:
Can you walk me through why this is expected for `closed='right'`? For some reason, this looks like `closed='left'` to me, but I'm probably missing something.

Reply:
Just wanted to enclose other options for closed argument. `left` and `neither` options are causing issues as you stated. Could be removed if unnecessary

Reply:
can you should the results of all 4 of these cases *in* the PR itself (you can update the top is good), just to get an easy visual on the results.

Reply:
@TomAugspurger Sorry, I just realize I misread your comment. @jreback @TomAugspurger  I tried to explain all 4 cases at the top of PR.